### PR TITLE
table comments

### PR DESCRIPTION
**Pull request by mhuseinov**
- added table comments to entities

### CODE REVIEW

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

File: services/apps/alcs/src/alcs/notice-of-intent/notice-of-intent-document/notice-of-intent-document.entity.ts [hunk 1]
```diff
@@ -21,7 +21,10 @@ export enum VISIBILITY_FLAG {
 }
 
 @Unique('OATS_NOI_UQ_DOCUMENTS', ['oatsDocumentId', 'oatsApplicationId'])
-@Entity()
+@Entity({
+  comment:
+    "Links NOI documents with the NOIs they're saved to and logs other attributes",
```
Comment (lstod) [inline services/apps/alcs/src/alcs/notice-of-intent/notice-of-intent-document/notice-of-intent-document.entity.ts:26]: also '

File: services/apps/alcs/src/alcs/application-decision/application-decision-document/application-decision-document.entity.ts [hunk 2]
```diff
@@ -11,7 +11,10 @@ import { Auditable } from '../../../common/entities/audit.entity';
 import { Document } from '../../../document/document.entity';
 import { ApplicationDecision } from '../application-decision.entity';
 
-@Entity()
+@Entity({
+  comment:
+    "Links application decision document with the decision it's saved to",
```
Comment (lstod) [inline services/apps/alcs/src/alcs/application-decision/application-decision-document/application-decision-document.entity.ts:16]: single or double '?

File: services/apps/alcs/src/alcs/card/card-history/card-history.entity.ts [hunk 3]
```diff
@@ -1,8 +1,11 @@
 import { Column, Entity, ManyToOne } from 'typeorm';
-import { Card } from '../card.entity';
 import { EntityHistory } from '../../../common/entities/history.entity';
+import { Card } from '../card.entity';
 
-@Entity()
+@Entity({
+  comment:
+    "History of card status i.e. the column history of the card's journey on boards",
```
Comment (lstod) [inline services/apps/alcs/src/alcs/card/card-history/card-history.entity.ts:7]: ' or ''

**Review by lstod — COMMENTED** [review]
inconsistency from double single quotes or single quotes

File: services/apps/alcs/src/alcs/application-decision/application-decision-outcome-type/application-decision-outcome-type.entity.ts [hunk 4]
```diff
@@ -1,5 +1,8 @@
 import { Entity } from 'typeorm';
 import { BaseCodeEntity } from '../../../common/entities/base.code.entity';
 
-@Entity()
+@Entity({
+  comment:
+    "Code table for the possible outcomes of the chair''s application decision review",
```
[the remark this answers is not in the document]
Comment (mhuseinov) [reply services/apps/alcs/src/alcs/application-decision/application-decision-outcome-type/application-decision-outcome-type.entity.ts:6]: This is a typescript, so it does not require the quotes to be escaped. In this case, it is a typo that came from typeorm migration that uses raw sql. I'll address that

**Pull request merged**